### PR TITLE
ui views: Table -> BfDsTable

### PR DESCRIPTION
**Pull request by mikesavior**
Summary: change refs of Table to BfDsTable so that traversing the dom with react dev tools is easier to do

Test Plan: TBD

### CODE REVIEW

File: infra/internalbf.com/client/components/InternalMediaList.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { React } from "deps.ts";
 import { Progress } from "packages/bfDs/Progress.tsx";
 import { BfDsButton } from "packages/bfDs/BfDsButton.tsx";
-import { Column, Table } from "packages/bfDs/Table.tsx";
-import { TableCell } from "packages/bfDs/TableCell.tsx";
+import { Column, BfDsTable } from "packages/bfDs/BfDsTable.tsx";
```
Comment (justicart) [inline infra/internalbf.com/client/components/InternalMediaList.tsx:4]: Column should be BfDsColumn

File: infra/internalbf.com/client/components/InternalMediaList.tsx [hunk 2]
```diff
@@ -14,19 +14,19 @@ const columns: Array<Column<DataType>> = [
   {
```
Comment (justicart) [inline infra/internalbf.com/client/components/InternalMediaList.tsx:14]: Column should be BfDsColumn

File: infra/internalbf.com/client/components/InternalProjectsList.tsx [hunk 3]
```diff
@@ -14,19 +14,19 @@ const columns: Array<Column<DataType>> = [
   {
```
Comment (justicart) [inline infra/internalbf.com/client/components/InternalProjectsList.tsx:14]: Make sure you catch these Column types... Column -> BfDsColumn

File: infra/internalbf.com/client/pages/ChangesPage.tsx [hunk 4]
```diff
@@ -11,18 +11,18 @@ type DataType = {
 };
 
 const columns: Array<Column<DataType>> = [
```
Comment (justicart) [inline infra/internalbf.com/client/pages/ChangesPage.tsx:13]: Column -> BfDsColumn

File: infra/internalbf.com/client/pages/IBfOrganizationsPage.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { React, ReactRelay } from "deps.ts";
 import { IBfFrame } from "infra/internalbf.com/client/components/IBfFrame.tsx";
-import { Columns, Table } from "packages/bfDs/Table.tsx";
-import { TableCell } from "packages/bfDs/TableCell.tsx";
+import { BfDsColumns, BfDsTable } from "packages/bfDs/BfDsTable.tsx";
```
Comment (justicart) [inline infra/internalbf.com/client/pages/IBfOrganizationsPage.tsx:3]: Any time you change these "Columns" or "Column" types up here, make sure you change where they are used below

File: packages/client/components/settings/Media.tsx [hunk 6]
```diff
@@ -54,17 +54,17 @@ export function Media({ settings$key }: Props) {
     {
```
Comment (justicart) [inline packages/client/components/settings/Media.tsx:54]: BfDsColumns

File: packages/client/components/settings/WatchFolderList.tsx [hunk 7]
```diff
@@ -76,22 +76,22 @@ export function WatchFolderList({ settings$key }: Props) {
     {
```
Comment (justicart) [inline packages/client/components/settings/WatchFolderList.tsx:76]: BfDsColumns

**Review by justicart — CHANGES_REQUESTED** [review]
Make sure to check the types Column and Columns where you changed the imports to BfDsColumn and BfDsColumns... they need to be changed in the code where they are used alos

File: infra/internalbf.com/client/pages/ClipChangesPage.tsx [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import { React } from "deps.ts";
-import { Table } from "packages/bfDs/Table.tsx";
-import { TableCell } from "packages/bfDs/TableCell.tsx";
+import { Table } from "packages/bfDs/BfDsTable.tsx";
```
Comment (randallb) [inline infra/internalbf.com/client/pages/ClipChangesPage.tsx:2]: This should be bfds table

**Review by randallb — DISMISSED** [review]

File: infra/internalbf.com/client/pages/ClipChangesPage.tsx (hunk 8 above)
Comment (mikesavior) [reply infra/internalbf.com/client/pages/ClipChangesPage.tsx]: np!

**Review by justicart — APPROVED** [review]

**Pull request merged**